src/builtins.c

```diff
@@ -524,6 +523,11 @@ JL_CALLABLE(jl_f_isdefined)
     }
     if (nargs != 2) {
         JL_NARGS(isdefined, 1, 1);
+        jl_depwarn("`isdefined(:symbol)` is deprecated, "
```

Comment:
add a note in deprecated.jl reminding to delete this when the other deprecations from this PR are removed